src/shared/src/types/change-request-types.ts

```diff
@@ -21,6 +22,8 @@ export interface ChangeRequest {
   implementedChanges?: ImplementedChange[];
   status: ChangeRequestStatus;
   requestedReviewers: User[];
+  projectProposedChanges?: ProjectProposedChanges;
+  workPackageProposedChanges?: WorkPackageProposedChanges;
```

Comment:
Just to confirm, do we not need the `WbsProposedChanges` field in the `ChangeRequest` shared type? Will the `WbsProposedChanges` type will be used somewhere else?

Reply:
nope! did it this way to make it easier from the frontend's perspective

Reply:
WbsProposedChanges is extended by work package and project proposed changes, so we still need it